src/utils/gitsha1.c.in

```diff
@@ -8,7 +8,7 @@
 
 // Autogenerated - Do not edit
 
-#include "gitsha1.h"
```

Comment:
This should be an absolute path, right? Now my compiler looks for `src/utils/utils/gitsha1.h`, which doesn't exist.